applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -183,6 +190,28 @@ static int get_input_report_idx(uint8_t report_id)
 	return -ENOENT;
 }
 
+static size_t get_output_report_idx(uint8_t report_id)
+{
+	for (size_t i = 0; i < ARRAY_SIZE(output_reports); i++) {
+		if (output_reports[i] == report_id) {
+			return i;
+		}
+	}
+	__ASSERT_NO_MSG(false);
+	return 0;
+}
+
+static size_t get_output_report_size(uint8_t report_id)
+{
```

Comment:
add the report size here - do it in all places where this function is called.
please don't add comment about one byte - make the code self documenting

```
size_t size = sizeof(report_id);

switch (report_id) {
...
}
```

Reply:
@MarekPieta do you agree with that? Earlier you have left the comment referring to that one byte in `get_output_report_size` function:
> In most usecases in nRF Desktop report size does not include 1 byte of report ID. I think we should keep it consistent here too.
Maybe the name of the function should be changed?

Reply:
report always include the report_id - this is how we send it in events so I don't think there will be any confusion.
BLE is the exception as at the step before sending there first byte is stripped.

Reply:
In general in nRF Desktop application the term `report size` so far refers to the size of the reports without including the report ID. Using similar convention here could be more consistent. Still, I will not NACK this PR because of using other convention here.

Reply:
This is not true. All variables in code that refer to the report use report_size that include both size of report_id and report data.
There is inconsistency in report headers that use REPORT_SIZE_WHATEVER instead of REPORT_DATA_SIZE_WHATEVER but I don't think this justifies creating a new term. If anything we could fix the inconsistency in headers but that could impact a lot of things so let's leave it as is for now.